### DISCUSSION TITLE
Add "skipCRDs" option to helmDefaults

### DISCUSSION THREAD

**Discussion by SeWieland** [helmfile/helmfile#929] [Ideas]
The [--skip-crds](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#method-1-let-helm-do-it-for-you) flag is very much needed if one wants to deploy CRDs seperately (not managed by the helm release).

As it is planned to [remove the --args flag from the helmfile command](https://github.com/helmfile/helmfile/blob/main/docs/proposals/towards-1.0.md#remove-the---args-flag-from-the-helmfile-command), an option is needed inside the helmDefaults configuration.

Proposal:

```yaml
helmDefaults:
  skipCRDs: false
```

**Comment by yxxhero** [comment]
Pr is welcome
